libethcore/ChainOperationParams.h

```diff
@@ -63,7 +63,7 @@ class PrecompiledContract
     u256 m_startingBlock = 0;
 };
 
-static constexpr int64_t c_infiniteBlockNumer = std::numeric_limits<int64_t>::max();
+static constexpr int64_t c_infiniteBlockNumber = std::numeric_limits<int64_t>::max();
```

Comment:
The `static` is redundant for `constexpr`.

Reply:
I thought that `static `and `constexpr `are orthogonal concepts? I agree that `static `doesn't make sense here (regardless of the constexpr) since this is a header file and `c_infiniteBlockNumber` is defined outside of a class - from what I understand, static when used this way limits the scope of the variable to the translation unit which means that each cpp file which includes the header will have a separate instance of c_infiniteBlockNumber (though each instance will have the same value) which seems unnecessary.

Reply:
I'm also confused now. On the language level that might be true. But also `constexpr` implies `inline` and `inline` implies `static`. 

I check than in practice the `constexpr` symbols are not exported.